### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by hannes-ucsc**

### CODE REVIEW

File: test/service/test_response.py [hunk 1]
```diff
@@ -38,7 +38,9 @@ def tearDownClass(cls):
         super().tearDownClass()
 
     def get_hits(self, entity_type: str, entity_id: str):
-        """Fetches hits from es instance searching for a particular entity ID"""
+        """
+        Fetches hits from es instance searching for a particular entity ID
```
Comment (danielsotirhos) [inline test/service/test_response.py]:
```suggestion
        Fetches hits from ES instance searching for a particular entity ID
```
or `Elasticsearch`

File: src/azul/service/responseobjects/elastic_request_builder.py [hunk 2]
```diff
@@ -542,11 +538,10 @@ def transform_request(self,
...
             list_adjustment = 1 if len(es_hits) > pagination['size'] else 0
             if 'search_before' in pagination:
-                hits = [x['_source'] for x in
-                        reversed(es_hits[0:len(es_hits) - list_adjustment])]
+                hits = reversed(es_hits[0:len(es_hits) - list_adjustment])
             else:
-                hits = [x['_source'] for x in es_hits[0:len(es_hits) - list_adjustment]]
-
+                hits = es_hits[0:len(es_hits) - list_adjustment]
+            hits = [hit['_source'] for hit in hits]
```
Comment (danielsotirhos) [inline src/azul/service/responseobjects/elastic_request_builder.py]:
```suggestion
            hits = [hit['_source'] for hit in es_hits[0:len(es_hits) - list_adjustment]]
            if 'search_before' in pagination:
                hits = reversed(hits)
```

File: test/service/test_response.py (hunk 1 above)
Comment (hannes-ucsc) [reply test/service/test_response.py]: Fixed.

File: src/azul/service/responseobjects/elastic_request_builder.py (hunk 2 above)
Comment (hannes-ucsc) [reply src/azul/service/responseobjects/elastic_request_builder.py]: As mentioned in stand-up, while I think distinguishing between hits and docs would made sense but that would be a rabbit hole I am not prepared to take on right now.

**Pull request merged**